recipe/meta.yaml

```diff
@@ -49,5 +49,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - Eric89GXL
+    - larsoner
```

Comment:
Please confirm this is correct, @larsoner.

Reply:
I added this because of https://github.com/conda-forge/staged-recipes/pull/2396#discussion_r100676519